ui/src/components/common/OverflowMenu.css

```diff
@@ -0,0 +1,41 @@
+.overflow-menu {
```

Comment:
This should be a scss file. 

I recommend making it a mixin then including itself at the bottom: 

```
@mixin overflow-menu {
   .overflow-menu { 
     ....
    }
}

@include overflow-menu;
```

then you can import `./OverflowMenu.scss` from inside the tsx file and the styles will still work. 

Eventually we'd move the `@include` higher up the hierarchy when we do a more complete sass refactoring